### ISSUE TITLE
False positive for gettext.install

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#396]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

The following code yields a false positive for undefined-variable.

```
#!python

import gettext
gettext.install("module")
_("test")
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/396
